src/components/add-class/AddClass.jsx

```diff
@@ -0,0 +1,138 @@
+          <label htmlFor="tutor_name" className="form-label ">
+            Tutor Name
+            <input
+              type="text"
+              className="form-control shadow  bg-white rounded"
+              id="tutor_name"
+              placeholder="eg. John Smith"
+              value={tutorName}
+              onChange={(e) => setTutorName(e.target.value)}
+            />
+          </label>
+        </div>
+        <div className="mb-3 animate__animated animate__fadeIn">
+          <label htmlFor="subject" className="form-label ">
+            Subject
+            <input
+              type="text"
+              className="form-control shadow  bg-white rounded"
+              id="subject"
+              placeholder="eg. Mathmatical Analysis"
+              value={subject}
+              onChange={(e) => setSubject(e.target.value)}
+            />
+          </label>
+        </div>
+        <div className="mb-3 animate__animated animate__fadeIn">
+          <label htmlFor="description" className="form-label ">
+            Description
+            <input
+              type="text"
+              className="form-control shadow  bg-white rounded"
+              id="description"
+              placeholder="Description"
+              value={description}
+              onChange={(e) => setDescription(e.target.value)}
+            />
+          </label>
+        </div>
+        <div className="mb-3 animate__animated animate__fadeIn">
+          <label htmlFor="price" className="form-label ">
+            Price
+            <input
+              type="number"
+              className="form-control shadow  bg-white rounded"
+              id="price"
+              placeholder="eg. $250"
+              value={price}
+              onChange={(e) => setPrice(e.target.value)}
+            />
+          </label>
+        </div>
+        <div className="mb-3 animate__animated animate__fadeIn">
+          <label htmlFor="duration" className="form-label ">
+            Duration
+            <input
+              type="number"
+              className="form-control shadow  bg-white rounded"
+              id="duration"
+              placeholder="eg. 50 minutes"
+              value={duration}
+              onChange={(e) => setDuration(e.target.value)}
+            />
+          </label>
+        </div>
+        <div className="mb-3 animate__animated animate__fadeIn">
+          <label htmlFor="image" className="form-label ">
+            image
+            <input
+              type="url"
+              className="form-control shadow  bg-white rounded"
+              id="image"
+              placeholder="eg. https://www.example.com/image.jpg"
+              value={image}
+              onChange={(e) => setImage(e.target.value)}
+            />
+          </label>
+        </div>
+        <button type="submit" className="btn-green ">
+          Add Class
+        </button>
+      </form>
+    </section>
```

Comment:
[OPTIONAL]
You are repeating yourself a lot here. Don't forget about the DRY principle. these input fields/labels can be abstracted as a subcomponent and then mapped with some sort of loop like `map`

example:


```
const formField = ({ id, type, placeholder, value, onChange }) => (
  <div className="mb-3 animate__animated animate__fadeIn">
    <label htmlFor={id} className="form-label ">
      {id.charAt(0).toUpperCase() + id.slice(1)}
      <input
        type={type}
        className="form-control shadow  bg-white rounded"
        id={id}
        placeholder={placeholder}
        value={value}
        onChange={onChange}
      />
    </label>
  </div>
);
```